conf/app.routes

```diff
@@ -1,4 +1,4 @@
 # microservice specific routes
 
-GET        /authorised/:utr         controllers.TrustAuthController.authorisedForUtr(utr: String)
+GET        /authorised/:utr         controllers.TrustAuthController.authorisedForIdentifier(utr: String)
```

Comment:
Could you also rename the argument in the endpoint to identifier.

Reply:
Updated